documentation/ag-grid-docs/src/components/example-grid/Example.tsx

```diff
@@ -0,0 +1,388 @@
+import { useDarkmode } from '@utils/hooks/useDarkmode';
+import { AgChartsEnterpriseModule } from 'ag-charts-enterprise';
+import { memo, useCallback, useEffect, useMemo, useRef, useState } from 'react';
+
+import type {
+    ColDef,
+    ColGroupDef,
+    CsvExportParams,
+    ExcelExportParams,
+    GridApi,
+    GridOptions,
+    GridReadyEvent,
+    SideBarDef,
+    Theme,
+} from 'ag-grid-community';
+import { AllCommunityModule, themeAlpine, themeBalham, themeMaterial, themeQuartz } from 'ag-grid-community';
+import {
+    CellSelectionModule,
+    ClipboardModule,
+    ColumnMenuModule,
+    ColumnsToolPanelModule,
+    ContextMenuModule,
+    ExcelExportModule,
+    IntegratedChartsModule,
+    MasterDetailModule,
+    MultiFilterModule,
+    NewFiltersToolPanelModule,
+    PivotModule,
+    RichSelectModule,
+    RowGroupingModule,
+    RowGroupingPanelModule,
+    RowNumbersModule,
+    SetFilterModule,
+    SideBarModule,
+    StatusBarModule,
+} from 'ag-grid-enterprise';
+import { AgGridReact } from 'ag-grid-react';
+
+import styles from './Example.module.scss';
+import { Toolbar } from './Toolbar';
+import { chartThemeOverrides, getDefaultChartThemes } from './config/chartOverrides';
+import {
+    autoGroupColDef,
+    columnTypes,
+    dataTypeDefinitions,
+    largeColCount,
+    largeDefaultCols,
+    smallColCount,
+    smallDefaultCols,
+} from './config/colDefs';
+import { excelStyles } from './config/excelStyles';
+import { COUNTRY_CODES, colNames, countries, createRowItem } from './data';
+import { createDataSizeValue, suppressColumnMoveAnimation } from './utils';
```

Comment:
You'll want to remove it here too

```suggestion
import { createDataSizeValue } from './utils';
```